daemon/cmd/daemon.go

```diff
@@ -112,6 +112,8 @@ type Daemon struct {
 	monitorAgent *monitoragent.Agent
 	ciliumHealth *health.CiliumHealth
 
+	deviceManager *DeviceManager
```

Comment:
A more suitable place for this might be in `pkg/datapath`, but I'd like to leave it here for now.